hack/lib/serverless.bash

```diff
@@ -49,6 +40,11 @@ function install_serverless_latest {
   logger.info "Installing latest version of Serverless..."
   deploy_serverless_operator_latest || return $?
 
+  # TODO: Remove this when the previous release has readiness checks available (since 1.12).
+  if versions.le "$(metadata.get olm.replaces)" 1.11.0; then
+    timeout 120 "[[ \$(oc get namespace ${SERVING_NAMESPACE} ${EVENTING_NAMESPACE} --no-headers | wc -l) != 2 ]]" || return $?
+  fi
```

Comment:
Isn't this the block where we effectively are at 1.12? (`install_serverless_latest`)

Reply:
Good catch! And sorry. I'm blind. Fixed.